packages/search-ui-app-search-connector/src/__tests__/buildResponseAdapterOptionsFromConfig.test.js

```diff
@@ -0,0 +1,71 @@
+import buildResponseAdapterOptionsFromConfig from "../buildResponseAdapterOptionsFromConfig";
+
+describe("buildResponseAdapterOptionsFromConfig", () => {
+  describe("additionalFacetValueFields", () => {
+    it("should extract additionalFacetValueFields for geo facets", () => {
+      expect(
+        buildResponseAdapterOptionsFromConfig({
+          facets: {
+            headquarters: {
+              center: "40.0374748, -76.3046049",
+              type: "range",
+              unit: "m",
+              ranges: [
+                { from: 0, to: 100, name: "Close" },
+                { from: 100, to: 500, name: "Far" }
+              ]
+            },
+            location: {
+              center: "40.0374748, -76.3046049",
+              type: "range",
+              unit: "mi",
+              ranges: [
+                { from: 0, to: 100, name: "Nearby" },
+                { from: 100, to: 500, name: "A longer drive." },
+                { from: 500, name: "Perhaps fly?" }
+              ]
+            },
+            acres: {
+              type: "range",
+              ranges: [
+                { from: -1, name: "Any" },
+                { from: 0, to: 1000, name: "Small" },
+                { from: 1001, to: 100000, name: "Medium" },
+                { from: 100001, name: "Large" }
+              ]
+            }
+          }
+        })
+      ).toEqual({
+        additionalFacetValueFields: {
+          headquarters: {
+            center: "40.0374748, -76.3046049",
+            unit: "m"
+          },
+          location: {
+            center: "40.0374748, -76.3046049",
+            unit: "mi"
+          }
+        }
+      });
+    });
+
+    it("should not extract additionalFacetValueFields if there are no geo fields", () => {
```

Comment:
Just to check about a semi-unrelated corner case:

Will there be any issues / do we want to 'test' what happens if someone passes in a facet obj that has a `center` but no `unit` field, or has `unit` but no `center`? Would elasticsearch still be able to parse that without issue, out of curiosity?

Reply:
For simplicity,  I think I am going to just assume them required for now. They are both required field for App Search, which will error with a 4xx if they are missing.

Reply:
Ahh okay! In that case, what do you think about changing [change line 4 here](https://github.com/elastic/search-ui/pull/260/files#diff-cce8ae85c9de219c2101c466711bcf99R4) to `facetConfig.unit && facetConfig.center` instead of `||`?

Reply:
Ah, yes, of course.